django_messages/views.py

```diff
@@ -121,7 +121,7 @@ def reply(request, message_id, form_class=ComposeForm,
         form = form_class(initial={
             'body': quote_helper(parent.sender, parent.body),
             'subject': subject_template % {'subject': parent.subject},
-            'recipient': [parent.sender,]
+            'recipient': [parent.recipient]
```

Comment:
Are you sure this is a correct change? This is the reply-view, so the user wants to send a reply to the message "parent", so the recipient is the sender of the "parent" message. 
If you disagree, please explain, what's wrong in the original code.

Reply:
Yes you are right, please fix it for me. While I was testing the app, I was writing message to my test account and I wanted to reply to myself. 
Thanks!